cmd/main.go

```diff
@@ -12,6 +12,7 @@ import (
 	"net"
 
 	pb "github.com/opiproject/opi-api/network/cloud/v1alpha1/gen/go"
```

Comment:
This needs to be removed right ?

Reply:
correct, once we complete transition to new APIs, old will be removed